HW1/P2/P2b.py

```diff
@@ -1,3 +1,26 @@
 from P2 import *
+from pyspark import SparkContext
+import random
 
-# Your code here
+sc = SparkContext("local", "Simple App")
+
+array_1 = [i for i in range(2000)]
+array_2 = [i for i in range(2000)]
+
+new_array = []
+for i in array_1:
+	for j in array_2:
+		new_array.append((j,i))
```

Comment:
You create the ranges and the cartesian product in python, but you should have done it in spark instead.